README.md

```diff
@@ -60,7 +60,13 @@ git config --global --add safe.directory [your dir here]
 
 ### Create a new version
 
-Create a tag on a commit with the following format `v{marjor}.{minor}.{patch}`; i.e.: v0.4.0; this will create a version accordingly.
+1. Go to the `Actions` tab of this repository.
+2. In the left side plane, click on `Terraform Provider Release Deployer` workflow.
```

Comment:
Where is this "terraform Provider Release Deployer" ? When I click on "Actions" tabs, all I see is : 
<img width="678" alt="image" src="https://user-images.githubusercontent.com/117275814/218423050-7a4502cf-6de9-4d0d-9f23-89c9d97f8c87.png">

I am missing something ?

Reply:
It should be there after the changes are merged as I've changed the name of the workflow.